packages/apollo-voyager-conflicts/src/strategies/index.ts

```diff
@@ -0,0 +1,17 @@
+import { ConflictResolutionStrategy } from '../api/ConflictResolutionStrategy'
+import { ObjectStateData } from '../api/ObjectStateData'
+
+export interface StrategyMap {[key: string]: ConflictResolutionStrategy}
```

Comment:
This seems strange to me. If we define object with set of strategies it will mean the same. We do not need interface for that.

Reply:
Fair enough, it was more so to strongly type the functions as ConflictResolutionStrategy types, however I can do that another way also